examples/graphviz_drawing/README.txt

```diff
@@ -0,0 +1,6 @@
+Graphviz Drawing
+----------------
+
+Examples using Graphviz for layout and drawing via `nx_agraph`.
+These examples need Graphviz and
+`PyGraphviz <https://pygraphviz.github.io/documentation.html>`_.
```

Comment:
This got changed to:
- https://pygraphviz.github.io/documentation/stable/

But, now that it has been updated, it may be better to point to:
- https://pygraphviz.github.io

Reply:
These are now handled via intersphinx.